lib/commands/general.js

```diff
@@ -109,11 +118,31 @@ commands.openNotifications = async function () {
 };
 
 commands.updateSettings = async function (settings) {
-  return await this.uiautomator2.jwproxy.command('/appium/settings', 'POST', {settings});
+  // we have some settings that are set on the settings object in the driver
+  // only, for example image finding settings. The uiauto2 server does not know
+  // what to do with them, so just set them on this driver's settings instance,
+  // and don't forward them to the server
+  let driverOnlySettings = {};
+  let serverSettings = {};
+  for (let setting of Object.keys(settings)) {
```

Comment:
_.keys

Reply:
why would we prefer lodash over the built-in language feature?

Reply:
As for my taste lodash wrappers are safer. They perform safe verification even if passed values are kinda broken

Reply:
fair enough. i also did a little research and we use `_.keys` about twice as much as `Object.keys` in our codebases, so might as well try and keep things consistent

Reply:
https://github.com/jashkenas/underscore/issues/149